### DISCUSSION TITLE
Only one time payments?

### DISCUSSION THREAD

**Discussion by Alexva397** [weareseeed/react-square-web-payments-sdk#37] [General]
Hey guys, I just had a quick question regarding payments. Your documentation states:

![Screen Shot 2022-03-29 at 09 05 39 (2)](https://user-images.githubusercontent.com/72521427/160629834-9b8b0bce-4f46-4a47-b879-8058f765df5b.png)

Does this mean I cannot use this package like my current usage of `react-square-payment-form`? Am I able to generate a token, not charge the card immediately and save the generated token for future use? If it is possible, is the token only usable for one payment?

Thanks!
Alex

**Comment by danestves** [comment]
Hi @Alexva397 instead of use this

```tsx
createVerificationDetails={() => ({
  ...details,
  // the rest of your verification details
  intent: 'CHARGE',
})}
```

You can change the `intent` to `STORE`:

```
intent: 'STORE',
```
